pkg/controller/disruption/disruption.go

```diff
@@ -452,9 +478,21 @@ func (dc *DisruptionController) trySync(pdb *policy.PodDisruptionBudget) error {
 		return err
 	}
 
-	currentHealthy := countHealthyPods(pods)
-	err = dc.updatePdbSpec(pdb, currentHealthy, desiredHealthy, expectedCount)
+	currentTime := time.Now()
+	disruptedPods, recheckTime := buildDisruptedPodMap(pods, pdb, currentTime)
```

Comment:
Yes, but on properly running cluster i expect the time to be set with ~1sec accuracy. Also the deleteTimestamp propagation should be done in 1-2 sec max. The controller has the tolerance up to 1 min.